addons/mail/static/src/models/rtc_session.js

```diff
@@ -323,6 +322,18 @@ registerModel({
                 { shadow: true },
             );
         },
+        /**
+         * Ensures that whether we download the videoStream is based on the
+         * presence of cards that should display that stream.
+         *
+         * @private
+         */
+        _onChangeCallParticipantCards() {
+            if (!this.channel || !this.channel.rtc || !this.rtcPeerConnection) {
+                return;
+            }
+            this.rtcPeerConnection.setVideoReceiverActivity(this.callParticipantCards && this.callParticipantCards.length > 0);
```

Comment:
Here param
`this.callParticipantCards && this.callParticipantCards.length > 0`
that becomes
`allowReception`
in signature.

Instead of passing param and encouraging flow-specific state, introduce a (computed) field here.